www/result.inc

```diff
@@ -175,7 +175,7 @@ $page_description = "Website performance test result$testLabel.";
                 if ($fvMedian)
                 {
                     if ($testResults->countRuns() > 1)
-                        echo '<h2>Performance Results (Median Run)</h2>';
+                        echo '<h2>Performance Results (Median Run - ' . $median_metric . ')</h2>';

```

Comment:
Just a small tweak to prevent XSS

```suggestion
                        echo '<h2>Performance Results (Median Run - ' . htmlspecialchars($median_metric) . ')</h2>';
```